Expensify/App
# Hide automated accounts from Group

@deetergp Please review this draft PR. 

### Details
- Added certain consts to create exclude list
- Changed the same for IOU options as well
- 

### Fixed Issues

$ https://github.com/Expensify/App/issues/4486

### Tests
1. Tested creating a new group on all platforms
2. Tested Recent option in the New Group
3. Removed Split Money and Request Money options for excluded emails
4. Removed Request Money on the Compose Button



### QA Steps
1. Go to Home screen and click on the Green FAB
2. Choose New Group
3. It shouldn't show up any excluded emails in the list or the recent options
4. Same has to be checked for Split Money and Request Money
5. In the 1:1 chat between Concierge or Chronos, ensure that the Request Money option is not shown



### Tested On


### Screenshots


#### Web
https://user-images.githubusercontent.com/3069065/130951687-a0dcd407-7f24-4ec3-b60d-196255b2dcfb.mp4

#### Mobile Web
https://user-images.githubusercontent.com/3069065/130951706-bfdebdf5-4247-4bfe-b49d-8964c1f3a84c.mp4


#### Desktop
https://user-images.githubusercontent.com/3069065/130951724-c7d54bb9-c261-4ca8-b4e4-9600b330a9c0.mp4

#### iOS
https://user-images.githubusercontent.com/3069065/130951756-3fd81db3-5bf6-4038-ab29-1888bbe3fa30.mp4

#### Android

https://user-images.githubusercontent.com/3069065/130956835-5fb8cc72-4851-4ac9-a9ea-a4eabfa413ab.mp4
